internal/app/apid/pkg/backend/apid_factory.go

```diff
@@ -6,26 +6,30 @@ package backend
 
 import (
 	"crypto/tls"
-	"sync"
 
+	"github.com/siderolabs/gen/containers"
 	"github.com/siderolabs/grpc-proxy/proxy"
-	"google.golang.org/grpc/credentials"
 )
 
 // APIDFactory caches connection to apid instances by target.
 //
 // TODO: need to clean up idle connections from time to time.
```

Comment:
is this todo still valid?

Reply:
yes, I think so. Not sure that we 'need', but definitely we don't clean up.